bindings/node/test/clientEncryption.test.js

```diff
@@ -178,7 +178,9 @@ describe('ClientEncryption', function () {
       expect(dataKey._bsontype).to.equal('Binary');
 
       // Remove and re-insert with a fixed UUID to guarantee consistent output
-      const doc = (await keyVaultColl.findOneAndDelete({ _id: dataKey })).value;
+      const doc = (
+        await keyVaultColl.findOneAndDelete({ _id: dataKey }, { writeConcern: { w: 'majority' } })
+      ).value;
       doc._id = new BSON.Binary(Buffer.alloc(16), 4);
       await keyVaultColl.insertOne(doc);
```

Comment:
This one is probably the one where it affects the rest of the test the most?

Reply:
Yeah I think this one is the most and the other is just 1.